src/Avalonia.Controls/Window.cs

```diff
@@ -282,6 +284,32 @@ public Task<TResult> ShowDialog<TResult>()
                 return result.Task;
             }
         }
+        
+        private IDisposable GetModal()
```

Comment:
This should probably be called something like `ShowModal` as its actually showing the modal dialog, not just getting it.